### DISCUSSION TITLE
Donate to you, thanks you

### DISCUSSION THREAD

**Discussion by vanhienne** [DeviaVir/zenbot#2666] [General]
I would like donate to you for thanks. Could you send me address of XRP.! Because XRP has low fee :) , Deviavir !

**Comment by DeviaVir** [comment]
Very nice of you, @vanhienne.

You can use `rDsbeomae4FXwgQTJp9Rs64Qg9vDiTCdBv` with destination tag `73192891`.

Thanks 🙏🏻

**Comment by vanhienne** [comment]
HI DEVIAVIR

I have just sent 49xrp for first, anh test. If you receive, please reply
me, thanks.



Vào 15:20, T.5, 25 Th3, 2021 Chase ***@***.***> đã viết:

**Reply by DeviaVir** [reply]
Received!

**Comment by vanhienne** [comment]
I had send 40 xrp to you! Thanks you very much.

Vào 17:06, T.5, 25 Th3, 2021 Chase ***@***.***> đã viết:

**Reply by DeviaVir** [reply]
🙏🏻  much appreciated @vanhienne - have a good day

**Comment by vanhienne** [comment]
I want to donate to Dastar81, how to contact with him? :)

Vào 22:13, T.3, 30 Th3, 2021 Chase ***@***.***> đã viết:

**Reply by DeviaVir** [reply]
I'm not sure who that is, where did you find this username?

**Comment by vanhienne** [comment]
Yes, who create the zenbot. :)

Vào 15:48, T.4, 31 Th3, 2021 Chase ***@***.***> đã viết:

**Reply by DeviaVir** [reply]
Ah, Carlos Rodriguez.

I'm afraid we only have  BTC address for him.

`187rmNSkSvehgcKpBunre6a5wA5hQQop6W`

**Comment by vanhienne** [comment]
Thanks you! Dou you have his email? If have, please to give me.

Vào 17:20, T.4, 31 Th3, 2021 Chase ***@***.***> đã viết:

**Comment by vanhienne** [comment]
Hi Deviavir
I monitor transactions, competitors can update prices continuously, how can
zenbot do like them . Please to advice me, many thanks.

Vào 08:47, T.2, 5 Th4, 2021 Nguyen Van Hien ***@***.***> đã viết:
